qa/evil-tests/src/test/java/org/elasticsearch/plugins/ListPluginsCommandTests.java

```diff
@@ -80,4 +88,38 @@ public void testTwoPlugins() throws Exception {
         assertTrue(output, output.contains("fake1"));
         assertTrue(output, output.contains("fake2"));
     }
+    
+    public void testPluginWithVerbose() throws Exception {
+        Environment env = createEnv();
+        Files.createDirectory(env.pluginsFile().resolve("fake1"));
+        Path configFile = Files.createFile(env.pluginsFile().resolve("fake1").resolve(PluginInfo.ES_PLUGIN_PROPERTIES));
+        List<String> config = Arrays.asList("description=fake plugin",
```

Comment:
There are other methods in InstallPluginCommandTests that could potentially be useful if we wanted to share them.